blazor/predefined-dialogs/customization-dialog-content.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Customization of Dialog Content in Blazor Predefined Dialogs Component | Syncfusion
+description: Checkout and learn here all about Draggable in Syncfusion Blazor Predefined Dialogs component and much more details.
+platform: Blazor
+control: Predefined Dialogs
+documentation: ug
+---
+
+# Customization of Dialog Content in Blazor Predefined Dialogs
+
+You can customize the predefined dialogs using the [DialogOptions.ChildContent](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogOptions.html#Syncfusion_Blazor_Popups_DialogOptions_ChildContent) property. Use the following code to render the textbox component inside `Prompt` dialog:
```

Comment:
What we can do with customization here? need to explain that. like we customize Dialog content or button or header.

Reply:
Detailed information provided now sir